.azure/app-cloud-e2e.yml

```diff
@@ -0,0 +1,144 @@
+      matrix:
+        'App: v0_app':
+          name: "v0_app"
+        'App: boring_app':
+          name: "boring_app"
+        'App: template_streamlit_ui':
+          name: "template_streamlit_ui"
+#        'App: template_react_ui':  # TODO: clarify visibility private/public
+#          name: "template_react_ui"
+        'App: template_jupyterlab':  # TODO: clarify where these files lives
+          name: "template_jupyterlab"
+        'App: idle_timeout':
+          name: "idle_timeout"
+        'App: collect_failures':
+          name: "collect_failures"
+        'App: custom_work_dependencies':
+          name: "custom_work_dependencies"
+        'App: drive':
+          name: "drive"
+        'App: payload':
+          name: "payload"
+        'App: commands_and_api':
+          name: "commands_and_api"
+    workspace:
+      clean: all
+    steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versi`onSpec: '3.8'
+
+    - script: |
+        ID=$(date +%s)
+        echo "##vso[task.setvariable variable=id;isOutput=true]$ID"
+      # TODO: parse the PR number
+      name: local
+
+    - task: Cache@2
+      inputs:
+        key: 'pip | "$(name)" | requirements/app/base.txt'
+        restoreKeys: |
+          pip | "$(Agent.OS)"
+        path: $(pip_cache_dir)
+      displayName: Cache pip
+
+    - bash: |
+        python --version
+        pip --version
+      displayName: 'Info'
+
+    - bash: python -m pip install -r requirements/app/devel.txt --quiet --find-links ${TORCH_URL}
+      env:
+        TORCH_URL: https://download.pytorch.org/whl/cpu/torch_stable.html
+      displayName: 'Install dependencies'
+
+    - bash: |
+        python -m pip install playwright
+        python -m playwright install --with-deps
+      displayName: 'Install Playwright system dependencies'
+
+    - bash: pip install -e .
+      displayName: 'Install lightning'
+
+    - bash: |
+        git clone https://github.com/Lightning-AI/LAI-lightning-template-jupyterlab-App examples/app_template_jupyterlab
+        cp examples/app_template_jupyterlab/tests/test_template_jupyterlab.py tests/tests_app_examples/test_template_jupyterlab.py
+      condition: eq(variables['name'], 'template_jupyterlab')
+      displayName: 'Clone Template Jupyter Lab Repo'
+
+    - bash: git clone https://github.com/Lightning-AI/lightning-template-react examples/app_template_react_ui
+      condition: eq(variables['name'], 'template_react_ui')
+      displayName: 'Clone Template React UI Repo'
+
+    - bash: |
+        mkdir -p ${VIDEO_LOCATION}
+        ls -l examples/${TEST_APP_NAME}
+        ls -l tests/tests_app_examples
+        python -m pytest tests/tests_app_examples/test_${TEST_APP_NAME}.py::test_${TEST_APP_NAME}_example_cloud --timeout=900 --capture=no -v --color=yes
+      env:
+        HEADLESS: "1"
+        PACKAGE_LIGHTNING: "1"
+        CLOUD: "1"
+        VIDEO_LOCATION: '$(Build.ArtifactStagingDirectory)/videos'
+        PR_NUMBER: $(variables['local.id'])
```

Comment:
Doesn't seem to be working as expected.
![Screenshot 2022-08-19 at 00 40 50](https://user-images.githubusercontent.com/7218339/185507730-658bba0c-43a5-4a13-bdc4-4cefcc2a6016.png)

Reply:
Shall be fixed now, mind checking again :)

Reply:
![Screenshot 2022-08-19 at 10 39 53](https://user-images.githubusercontent.com/7218339/185580127-c56a75e6-cdb6-4946-978d-03415dec522a.png)

not yet

Reply:
ok, where do you see it?

Reply:
You can also see it in the logs: https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=90561&view=logs&j=d86e5e1d-9f7a-5f82-a0c9-907939645ce1&t=24443d46-bbe8-562c-694f-467a1ad49ac3&l=49